GridTariffAPI.lib/Services/Helpers/ServiceHelper.cs

```diff
@@ -103,5 +103,22 @@ public DateTimeOffset DbTimeZoneDateToUtc(DateTime dateTime)
         {
             return new DateTimeOffset(dateTime, _gridTariffApiConfig.TimeZoneForQueries.GetUtcOffset(dateTime));
         }
+
+        public virtual bool TimePeriodIsIncludingLocaleToday(DateTimeOffset fromDateTime, DateTimeOffset toDateTime)
+        {
+            var localNow = ToConfiguredTimeZone(DateTimeOffset.UtcNow);
+            var localTodayStart = new DateTimeOffset(localNow.Date, localNow.Offset);
+            var localTodayEnd = localTodayStart.AddDays(1);
+
+            if (localTodayStart >= toDateTime)
```

Comment:
Skal vi vurdere om 00:00:00 er "i dag"? Og la 23:59:59... være i dag, men morgendagens 00:00:00 være i morgen? Og/eller har vi noe fra/til med eller ikke med-semantikk fra før?

Reply:
Litt usikker på hva du mener.
Jeg tenker at spørring hvor spørring.tiltidspunkt == midnatt i dag (00:00) ikke er innenfor.
Tilsvarende tenker jeg at spørring.fratidspunkt == midnatt i morgen (00:00) ikke er innenfor.

Det blir vel da "fra-og-med til" for både spørring og for "i dag".

Reply:
resolved ref samtale.